src/Features/CSharp/Portable/Completion/CompletionProviders/SymbolCompletionProvider.cs

```diff
@@ -74,6 +142,35 @@ protected override async Task<bool> IsSemanticTriggerCharacterAsync(Document doc
             return token.Kind() != SyntaxKind.NumericLiteralToken;
         }
 
+        /// <returns><see langword="null"/> if not an argument list character, otherwise whether the trigger is in an argument list.</returns>
```

Comment:
couldn't we make this only be called if we sholud trigger and we have an arg list char?  in that case, this would just be a true/false question.  seems simpelr to me.

Reply:
If you were indoctrinated by the way `IsTriggerOnDotAsync` works, then this approach "makes sense." I like having the consistent approach between the two, personally.

Reply:
SGTM.